zee/config/config.ron

```diff
@@ -1,11 +1,22 @@
 (
-    // Which theme to use
+    //
+    // GENERAL SETTINGS
+    // ================
+    //
+
+    // Which colour theme to use.
     theme: "zee-gruvbox",
 
-    // Modes and tree sitter parsers
+
+
+    //
+    // MODES AND TREE-SITTER PARSERS
```

Comment:
This is still a multi-line comment introducing a completely different section of the configuration file.  I would like to suggest this format as it still is a visual indication of the new section to begin but easier to edit than before.

What do you think, @mcobzarenco?

Reply:
:+1: Happy with this, thanks for the changes!